websites/Z/ZARP Gaming/presence.ts

```diff
@@ -0,0 +1,119 @@
+const presence = new Presence({
+	clientId: "990710812320612432",
+});
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "zarp-512x512",
+	};
+	switch (window.location.host) {
+		case "zarpgaming.com":
+			switch (window.location.pathname) {
+				case "/index.php":
+					presenceData.details = "Viewing the home page";
+					break;
+				case "/index.php/user-list":
+					presenceData.details = "Viewing the user list";
+					break;
+				case "/index.php/zarp-vip":
+					presenceData.details = "Viewing the VIP page";
+					break;
+				case "/index.php/zarp-vip/booster-pack":
+					presenceData.details = "Viewing the booster packs";
+					break;
+				case "/index.php/help":
+					presenceData.details = "Viewing the help page";
+					break;
+				case "/index.php/help/faq":
+					presenceData.details = "Viewing the FAQ";
+					if (window.location.pathname.includes("faq/"))
+						presenceData.state = document.title;
+					break;
+				case "/index.php/profile/profile-edit":
+					presenceData.details = "Editing profile";
+					break;
+				case "/index.php/2013-11-13-21-35-14/":
```

Comment:
This doesn't seem right, why would you support a 2013 page and not others?

Reply:
This is just show the forums is, it is correct and all tested.

Reply:
so everything on the forums is on this exact page with nothing after it? Show me a screenshot of that